### DISCUSSION TITLE
OrientDB configuration for ensuring data reaches the disk

### DISCUSSION THREAD

**Discussion by pa-elliotmeisinger** [orientechnologies/orientdb#10244] [Q&A]
We are currently using  v.3.2.11 but are looking to upgrade to latest. We are trying to use a disk snapshot backup strategy and need to ensure that OrientDB is persisting to disk often enough. We we don't want to go through the process of freezing databases to achieve quiescence of some level. 

We've had dataloss and some corruption occur when a machine is restarted for updates or the process fails otherwise. Our hypothesis is that OrientDB was caching data and hadn't written it to disk.

I've read though the [Server Configuration ](https://orientdb.com/docs/3.2.x/admin/Configuration.html) and the [PLocal Storage](https://orientdb.com/docs/3.2.x/internals/Paginated-Local-Storage.html) pages. I may be missing something or misinterpreting some of the configuration settings or how they may interact or overlap. I would appreciate any feedback from the community on how to ensure that we are flushing or committing journal/write ahead logs completely, often enough. Performance considerations are important but we will accept some drop in performance to write more often.

I see the setting [storage.diskCache.writeCachePageTTL](https://orientdb.com/docs/3.2.x/admin/Configuration.html#storagediskcachewritecachepagettl) and that the default is set to 86400 seconds (24 hours). I thought that turning this down to something smaller may be the appropriate action. At the same time I see other settings which by the name and description, I think would provide the same effect [storage.diskCache.writeCachePageFlushInterval](https://orientdb.com/docs/3.2.x/admin/Configuration.html#storagediskcachewritecachepageflushinterval).

In summary, we want OrientDB to ensure that it has at least written data to the disk every hour or so. Any help being certain we have configured the service to achieve this is greatly appreciated.
